tcms/testplans/templates/testplans/mutable.html

```diff
@@ -92,12 +97,63 @@
                 </div>
             </div>
 
+            <div class="form-group">
+                <div class="panel">
+                    <div class="panel-heading"> {% trans "Notify: " %} </div>
+                    <div class="col-md-1 col-lg-1">
+                        <label> {% trans "Author" %} </label>
+                    </div>
+                    <div class="col-md-3 col-lg-3">
+                        <input class="bootstrap-switch" name="auto_to_plan_author" type="checkbox" {% if form.auto_to_plan_author.value %} checked {% endif %}>
+                    </div>
+                    <div class="col-md-1 col-lg-1">
+                        <label> {% trans "TestCase author" %} </label>
+                    </div>
+                    <div class="col-md-3 col-lg-3">
+                        <input class="bootstrap-switch" name="auto_to_case_owner" type="checkbox" {% if form.auto_to_case_owner.value %} checked {% endif %}>
+                    </div>
+                    <div class="col-md-1 col-lg-1">
+                        <label> {% trans "Default tester" %} </label>
+                    </div>
+                    <div class="col-md-3 col-lg-3">
+                        <input class="bootstrap-switch" name="auto_to_case_default_tester" type="checkbox" {% if form.auto_to_case_default_tester.value %} checked {% endif %}>
+                    </div>
+                </div>
```

Comment:
These still sit a bit misaligned, this is because panels have inner padding and margin and yours isn't sized properly.

The easiest way is to:
1) Remove `class="panel"`
2) Move the existing panel heading directly under form-group and make it a label. See the document label for sizing.
3) The rest of the widgets in this group are good as they are.